daisy/workflow/test.workflow

```diff
@@ -51,30 +51,30 @@
         }
       ]
     },
-    "postinstall stopped": {
+    "postinstall-stopped": {
       "waitForInstancesSignal": [{"name": "postinstall", "stopped": true}]
     },
-    "create image": {
+    "create-image": {
       "createImages": [
         {
           "name": "image-from-disk",
           "sourceDisk": "image"
         }
       ]
     },
-    "sub workflow": {
+    "sub-workflow": {
       "subWorkflow": {
         "path": "./test_sub.workflow"
       }
     }
   },
   "dependencies": {
-    "create disks": [],
```

Comment:
I'm not 100% on enforcing step names. I know you have dreams of doing per-step namespacing, but I'm not sold on it yet. Your call.